plugins/org.jboss.tools.jmx.remoting1_1_3/build.properties

```diff
@@ -0,0 +1,16 @@
+source.. = src/
+output.. = bin/
+bin.includes = META-INF/,\
+               .,\
+               lib/jboss-logging-3.1.4.GA.jar,\
+               lib/jboss-logging-processor-1.1.0.Final.jar,\
+               lib/jboss-logmanager-1.5.3.Final.jar,\
+               lib/jboss-marshalling-1.4.8.Final.jar,\
+               lib/jboss-marshalling-river-1.4.8.Final.jar,\
+               lib/jboss-remoting-3.3.3.Final.jar,\
+               lib/jboss-sasl-1.0.4.Final.jar,\
+               lib/jdeparser-1.0.0.Final.jar,\
+               lib/remoting-jmx-1.1.3.Final.jar,\
+               lib/xnio-api-3.0.10.GA.jar,\
+               lib/xnio-nio-3.0.10.GA.jar,\
+               plugin.xml

```

Comment:
dont think that we should commit all these binaries since they should (and jmx already is) fetched and copied via maven dependency plugin